src/views/clusteroverview/SettingsTab/ClusterTab/components/EnablePreviewFeaturesSection/EnablePreviewFeaturesSection.tsx

```diff
@@ -10,14 +12,32 @@ import { Stack, StackItem } from '@patternfly/react-core';
 
 import ExpandSection from '../../../ExpandSection/ExpandSection';
 
+import PreviewFeaturesPopover from './PreviewFeaturesPopover';
+
 const EnablePreviewFeaturesSection: FC = () => {
   const { t } = useKubevirtTranslation();
 
   return (
-    <ExpandSection toggleText={t('Preview features')}>
+    <ExpandSection
+      toggleContent={
+        <>
+          {t('Preview features')}
```

Comment:
this can be inside the component line below?

Reply:
I tried it last week. It either won't toggle or won't show the popover. I can't remember which.